src/llrb-tree/LlrbTree.ts

```diff
@@ -175,6 +175,151 @@ export class LlrbTree<K, V> implements SonicMap<K, V, LlrbNode<K, V>> {
     }
   }
 
+  // Helper methods for LLRB deletion
+  private _isRed(node: LlrbNode<K, V> | undefined): boolean {
+    return node ? !node.b : false;
```

Comment:
Inline most basic checks, for performance. Performance is the number one priority in this library.